examples/darwin-framework-tool/commands/clusters/ClusterCommandBridge.h

```diff
@@ -73,10 +73,14 @@ class ClusterCommand : public ModelCommand {
         uint16_t __block responsesNeeded = repeatCount;
         dispatch_queue_t callbackQueue = dispatch_queue_create("com.chip.command", DISPATCH_QUEUE_SERIAL_WITH_AUTORELEASE_POOL);
 
+        __block auto * endpoint = [NSNumber numberWithUnsignedShort:endpointId];
+        __block auto * cluster = [NSNumber numberWithUnsignedInteger:clusterId];
+        __block auto * command = [NSNumber numberWithUnsignedInteger:commandId];
```

Comment:
Why do these need to be `__block`?  They don't get mutated, right?

Also, any reason to not use `@(endpointId)` etc to box?